api/src/main/java/com/github/streamshub/console/api/security/OidcTenantConfigResolver.java

```diff
@@ -40,6 +63,44 @@ void initialize() {
         if (oidc.getIssuer() != null) {
             oidcConfig.getToken().setIssuer(oidc.getIssuer());
         }
+
+        getTlsConfiguration().map(TlsConfiguration::getTrustStore).ifPresentOrElse(
+            this::configureTruststore,
+            () -> logger.infof("No truststore configured for OIDC provider")
+        );
+    }
+
+    Optional<TlsConfiguration> getTlsConfiguration() {
+        String dotSeparatedSource = "oidc.provider.trust";
+        String dashSeparatedSource = "oidc-provider-trust";
+        return tlsRegistry.get(dotSeparatedSource).or(() -> tlsRegistry.get(dashSeparatedSource));
+    }
+
+    void configureTruststore(KeyStore truststore) {
```

Comment:
The OIDC subsystem takes the path to a truststore, so we need to write the one from the TLS registry to a working file to provide to OIDC. This should no longer be necessary in the next Quarkus LTS where OIDC is aware of the TLS registry.

Reply:
Might be worth adding that as comment.

Reply:
Moved the original comment to the method's JavaDoc
> The OIDC subsystem takes the path to a truststore, so we need to write the one from the TLS registry to a working file to provide to OIDC. This should no longer be necessary in the next Quarkus LTS where OIDC is aware of the TLS registry.